src/control-plane/cloudfront-s3-portal.ts

```diff
@@ -324,7 +324,7 @@ export class CloudFrontS3Portal extends Construct {
           responseHeadersPolicy: responseHeadersPolicy,
         },
         defaultRootObject: 'index.html',
-        priceClass: PriceClass.PRICE_CLASS_ALL,
+        priceClass: PriceClass.PRICE_CLASS_100,
```

Comment:
Looks like 200 level is better to cover Asia.

https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/PriceClass.html